unitTests/ui/MLFacet/MLFacetTest.ts

```diff
@@ -0,0 +1,184 @@
+import * as Mock from '../../MockEnvironment';
+import { MLFacet } from '../../../src/ui/MLFacet/MLFacet';
+import { IMLFacetOptions } from '../../../src/ui/MLFacet/MLFacetOptions';
+import { IMLFacetValue } from '../../../src/ui/MLFacet/MLFacetValues/MLFacetValue';
+import { MLFacetTestUtils } from './MLFacetTestUtils';
+import { $$ } from '../../../src/Core';
+
+export function MLFacetTest() {
```

Comment:
Good job on these tests 👍

Reply:
Thanks! It's only a start, much more features to come